src/js/media-controller.js

```diff
@@ -1154,6 +1134,33 @@ const getShowingCaptionTracks = (controller) => {
   });
 };
 
+const getStreamType = (controller) => {
+  const { media } = controller;
+  
+  if (!media) return undefined;
```

Comment:
Previously, this returned `null`, why the change?

Reply:
Just for consistency with the spec. If there's no slotted media, I'm treating this as equivalent to "unimplemented". From "the out side" (aka the attribute value), this should be identical regardless.

Reply:
Interesting to look at this from either the media element or media chrome perspective. I could see this being "media chrome knows what stream type is, but it's unknown". But I'm good with matching the spec here.

Reply:
@heff yup was mulling on the same thing and this is where I (soft) landed. Open to rethinking that moving forward for this kinda stuff if we wanted to discuss in one of our weekly meetings.